src/content/blog/2025-01-22-eslint-2024-year-review.md

```diff
@@ -0,0 +1,156 @@
+| Cybozu                         | $4,200.00               |
+| Canva                          | $3,000.00               |
+| Sanity                         | $2,400.00               |
+| Icons8                         | $2,400.00               |
+| Discord                        | $2,400.00               |
+| Anagram Solver                 | $2,400.00               |
+| Nx                             | $2,400.00               |
+| HeroCoders                     | $2,400.00               |
+| SERP Triumph                   | $2,000.00               |
+| Principal Financial Group      | $1,800.00               |
+| (redacted)                     | $1,741.94               |
+| Nextbase Starter Kit           | $1,612.48               |
+| Ignition                       | $1,600.00               |
+| notion                         | $1,400.00               |
+| Bitwarden                      | $1,048.00               |
+| Siemens                        | $1,000.00               |
+
+**Note:** The redacted sponsor name is from an account that was of questionable origin and was eventually blocked by GitHub. However, the money did end up in our account so we are including it in the table for transparency.
+
+### Expenses review
+
+![2024 Expenses - Summarized in following table](/assets/images/blog/2025/2024-expenses.svg)
+
+In 2024, we spent $247,669.54 USD on the ESLint project as a whole. This includes regular maintenance and development as well as other costs related to the project. This is in comparison to the $187,964.30 of income for a **net loss of -$59,705.24**. See the table below for an overview.
+
+
+| **Category**                | **2024 Amount**  | **2023 Amount** | **Gain/Loss %** |
+|-----------------------------|------------------|-----------------|-----------------|
+| Maintenance and Development | $147,273.43      | $66,910.37      | +120.12%        |
+| TSC Stipend (Tidelift)      | $18,449.16       | $30,077.84      | -38.68%         |
+| Triage and Support          | $3,985.50        | $10,307.78      | -61.33%         |
+| Technical Writing           | $0.00            | $8,155.00       | -100.00%        |
+| Contributor Pool            | $14,100.30       | $4,500.00       | +213.34%        |
+| Payment Processor Fees      | $3,808.97        | $2,854.51       | +33.45%         |
+| Health Insurance            | $12,698.40       | $13,447.41      | -5.57%          |
+| Open Source Collective Fee  | $11,696.69       | $9,206.88       | +27.04%         |
+| Ecosystem/Dependencies      | $16,100.00       | $24,400.00      | -33.99%         |
+| Community Management        | $9,825.03        | $15,599.84      | -37.04%         |
+| Google Workspace            | $182.06          | $187.35         | -2.82%          |
+| Code Explorer               | $7,200.00        | $0.00           | +100%           |
+| Config Inspector            | $2,350.00        | $0.00           | +100%           |
```

Comment:
$0 + 100% would be still $0. Maybe it's better to leave the Gain/Loss field empty for those two entries?

Reply:
Technically it's +Infinity, but that seems silly to put in the table. I also don't think leaving the column empty makes sense because you lose the ability to quickly scan down the column to see the change.

Given that going from $2,350 to $0 would be -100%, I thought +100% would at least give a clear understanding that there was a positive change even if not technically mathematically correct.